lib/internal/url.js

```diff
@@ -266,18 +259,22 @@ class URL {
 }
 
 Object.defineProperties(URL.prototype, {
-  toString: {
-    // https://heycam.github.io/webidl/#es-stringifier
-    writable: true,
-    enumerable: true,
-    configurable: true,
+  [kFormat]: {
+    enumerable: false,
+    configurable: false,
```

Comment:
Yes, I know they are the default. I prefer to be explicit. Also, because this is an internal only method, I prefer to keep these values